csharp/src/Ice/Communicator.cs

```diff
@@ -1316,6 +1340,28 @@ private void AddAllAdminFacets()
             }
         }
 
+        private static Type? FindType(string csharpId)
```

Comment:
What's the point of the caching into _typeTable?

I can't see a situation where we'll find something in the _typeTable cache.

Reply:
This cache this is a leftover of the 3.7 logic where we cache the types and use the type Activator to create a new instance, but no longer required.